src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    data-qa="big"
+    >
+      <div class="input_container_big">
+        <img src="images/Search.svg"
+        alt="img"
+        class="big_img">
+        <input type="text" 
+        name="big" 
+        placeholder="Try &quot;Los Angeles&quot"
+        class="big_search search"
+        data-qa="keypress"
+        >
+      </div>
```

Comment:
```suggestion
      <div class="input_container_big">
        <img
          src="images/Search.svg"
          alt="img"
          class="big_img"
        >
        
        <input
          type="text" 
          name="big" 
          placeholder="Try &quot;Los Angeles&quot"
          class="big_search search"
          data-qa="keypress"
        >
      </div>
```

check your code style
fix all such cases
<img width="906" alt="image" src="https://user-images.githubusercontent.com/102411130/224638162-a834b2e1-7642-4079-8fe2-421b3d85dea3.png">
also, add blanks between multiline sibling elements